### PR TITLE
chore(curriculum): add quotes around strings in cypher project

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Related to #51036

### CODE REVIEW

**Review by Dario-DC — APPROVED** [review]
LGTM 🎉

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6551eebe6cbb2e6cadf9b468.md
Comment (zairahira) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6551eebe6cbb2e6cadf9b468.md:1]: Line #18 

```diff
- Delete your `number` variable and its value. Then, declare another variable called `text` and assign the string `Hello World` to this variable.
+ Delete your `number` variable and its value. Then, declare another variable called `text` and assign the string `'Hello World'` to this variable.
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/65688b5a1655a7a6caede847.md
Comment (zairahira) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/65688b5a1655a7a6caede847.md:1]: Line #28 

```diff
- You should reassign `text` the string `Albatross`. Do not modify `text = 'Hello World'`.
+ You should reassign `text` the string `'Albatross'`. Do not modify `text = 'Hello World'`.
```

**Review by zairahira — CHANGES_REQUESTED** [review]
As some of my suggestions are outside the scope of the changed files, I have added comments to the individual files.

Also, have a look at this change too:

Step 33

Line #23  

```diff
- You should still have a `text` variable with the value `Hello World`.
+ You should still have a `text` variable with the value `'Hello World'`.
```

**Review by zairahira — APPROVED** [review]

**Pull request merged**